distributed/comm/ws.py

```diff
@@ -378,7 +383,7 @@ async def connect(self, address, deserialize=True, **connection_args):
         kwargs = self._get_connect_args(**connection_args)
         try:
             request = HTTPRequest(f"{self.prefix}{address}", **kwargs)
-            sock = await websocket_connect(request, max_message_size=10_000_000_000)
+            sock = await websocket_connect(request, max_message_size=MAX_MESSAGE_SIZE)
```

Comment:
this should use the property you introduced, shouldn't it?

Reply:
@fjetter sorry I'm not sure I follow your question. 
The property is to allow the server connections to receive messages > 10mb. This line is to allow the client connections to receive messages >10Mb. AFAIK the two classes are disconnected.

Reply:
Maybe another way to say it is, where is the `max_message_size` `@property` that's introduced used? Doing a quick `git grep max_message_size` I see the property being defined here, but not actually used anywhere.

Reply:
Ah, it appears we're overwriting the [`max_message_size` in tornado's `WebSocketHandler`](https://github.com/tornadoweb/tornado/blob/1db5b45918da8303d2c6958ee03dbbd5dc2709e9/tornado/websocket.py#L298-L309)

Reply:
Just to be clear here, the previous limit was 10GB, not 10MB.  

If we need to go above this we could also just bump that limit to 100GB or something.  If it needs to be configurable then let's use dask's config system.

Reply:
@mrocklin as @jrbourbeau pointed out, we are overriding a Tornado WebSocketHandler property that was defaulting to 10Mb. To your point, the client connections on the other hand could accept much larger messages. 
I don't have strong opinions on making that value configurable or not. I will say it's a bit misleading that the property goes through `self.settings` when in fact, that's just a pass-through to the `Application` kwargs.